app/[locale]/institute/sections/health-centre/page.tsx

```diff
@@ -13,5 +26,631 @@ export default async function HealthCentre({
+          <Table>
+            <TableHeader>
+              <TableRow>
+                <TableHead>{text.timings.day}</TableHead>
+                <TableHead>Timings</TableHead>
+              </TableRow>
+            </TableHeader>
+            <TableBody>
+              {timings.days.map((entry, idx) => (
+                <TableRow key={idx}>
+                  <TableCell>{entry.day}</TableCell>
+                  <TableCell>
+                    {entry.timings.map((timing, idx) => (
+                      <div key={idx}>{`${timing.from} - ${timing.to}`}</div>
+                    ))}
+                  </TableCell>
+                </TableRow>
+              ))}
+            </TableBody>
+          </Table>
+        </Suspense>
+      </section>
+      <section className="container">
+        <Heading
+          glyphDirection="rtl"
+          heading="h3"
+          href={'#staff'}
+          id="staff"
+          text={text.headings.staff.toUpperCase()}
+        />
+        <Suspense fallback={<Loading />}>
+          <h3 className="my-10">{text.staff.officers}</h3>
+          <Table>
+            <TableHeader>
+              <TableRow>
+                <TableHead>{text.staff.sr}</TableHead>
+                <TableHead>{text.staff.name}</TableHead>
+                <TableHead>{text.staff.designation}</TableHead>
+                <TableHead>{text.staff.phone}</TableHead>
+              </TableRow>
+            </TableHeader>
+            <TableBody>
+              {medicalOfficersData.map((entry, idx) => (
+                <TableRow key={idx}>
+                  <TableCell>{entry.srNo}</TableCell>
+                  <TableCell>{entry.name}</TableCell>
+                  <TableCell>{entry.role}</TableCell>
+                  <TableCell>{entry.tel}</TableCell>
+                </TableRow>
+              ))}
+            </TableBody>
+          </Table>
+        </Suspense>
+        <Suspense fallback={<Loading />}>
+          <h3 className="my-10">{text.staff.other}</h3>
+          <Table>
+            <TableHeader>
+              <TableRow>
+                <TableHead>{text.staff.sr}</TableHead>
+                <TableHead>{text.staff.name}</TableHead>
+                <TableHead>{text.staff.designation}</TableHead>
+                <TableHead>{text.staff.phone}</TableHead>
+              </TableRow>
+            </TableHeader>
+            <TableBody>
+              {medicalStaffData.map((entry, idx) => (
+                <TableRow key={idx}>
+                  <TableCell>{entry.srNo}</TableCell>
+                  <TableCell>{entry.name}</TableCell>
+                  <TableCell>{entry.designation}</TableCell>
+                  <TableCell>{entry.phone}</TableCell>
+                </TableRow>
+              ))}
+            </TableBody>
+          </Table>
+        </Suspense>
+      </section>
+      <section className="container">
+        <Heading
+          glyphDirection="rtl"
+          heading="h3"
+          href={'#facilities'}
```

Comment:
We won't use this syntax for strings as it's unnecessarily verbose.
```suggestion
          href="#facilities"
```